apps/sms/views/shared/js/activity_handler.js

```diff
@@ -366,7 +364,7 @@ var ActivityHandler = {
               break;
             }
 
-            return text || formatValue('mms-message');
```

Comment:
it's not binding into our retranslation system. Setting data-l10n-id='mms-message' will do exactly the same but will react to retranslations.